src/containers/LearningpathPage/LearningpathPage.jsx

```diff
@@ -67,7 +68,14 @@ class LearningpathPage extends Component {
     const longName = getLongNameFromFilters(locale, location, subject);
 
     return getAllDimensions(
-      { subject, topicPath, learningpath, learningstep, filter: longName },
+      {
+        subject,
+        relevance,
+        topicPath,
+        learningpath,
+        learningstep,
+        filter: longName,
+      },
```

Comment:
sorter gjerne lister som dette alfabetisk når du endrer i de, gjør det mer oversiktlig for neste mann/dame